project/src/js/ui/round/view/button.js

```diff
@@ -298,5 +298,29 @@ export default {
           () => window.plugins.toast.show(i18n('notes'), 'short', 'bottom')
         )} />
     );
+  },
+  returnToTournament: function(ctrl) {
+    function handler() {
+      ctrl.hideActions();
+      m.route('/tournament/' + ctrl.data.tournament);
+    }
+    return (
+      <button key="returnToTournament" config={helper.ontouch(handler)}>
+        <span className="fa fa-trophy" />
+        Return to Tournament
+      </button>
+    );
+  },
+  withdrawFromTournament: function(ctrl) {
```

Comment:
This button does not belong to game. I prefer to stick to what is done in the website. Thus the withdraw button should be in the tournament screen only. You can then take the i18n message `withdraw` which is then self explanatory.